src/components/CourtOrderPoa/CourtOrderPoa.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <v-card flat class="mt-4 pa-8">
+    <v-row no-gutters align="start">
+      <v-col cols="3">
+        <label>Court Order <br>Number</label>
+      </v-col>
+      <v-col cols="9">
+        <v-form ref="courtNumRef" id="court-num-form" v-model="valid">
+          <v-text-field id="court-order-number-input"
+                        v-model="courtOrderNumber"
+                        label="Court Order Number"
+                        :rules="courtOrderNumRules"
+                        filled
+          />
+        </v-form>
+      </v-col>
+    </v-row>
+    <v-row no-gutters align="end">
+      <v-col cols="3">
+        <label>Plan of <br>Arrangement</label>
+      </v-col>
+      <v-col cols="9">
+        <v-checkbox id="plan-of-arrangement-checkbox"
+                    v-model="planOfArrangement"
+                    label="This filing is pursuant to a Plan of Arrangement"
+        />
+      </v-col>
+    </v-row>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+import { FormIF } from '@bcrs-shared-components/interfaces'
+
+@Component({})
+export default class CourtOrderPoa extends Vue {
+  // Refs
+  $refs!: Vue['$refs'] & {
+    courtNumRef: FormIF
+  }
+
+  /** Prompt the validations. Used for global validations. */
+  @Prop({ default: false })
+  private validate
+
+  // Local properties
+  private courtOrderNumber = ''
+  private courtOrderNumRules = []
+  private planOfArrangement = false
+  private valid = false
+
+  @Watch('validate')
+  validateCourtNum (): void {
+    if (this.planOfArrangement) {
+      // Apply TextField rules
+      this.courtOrderNumRules = [
+        (v: string) => !!v || 'A Court Order number is required',
+        (v: string) => !/^\s/g.test(v) || 'Invalid spaces', // leading spaces
+        (v: string) => !/\s$/g.test(v) || 'Invalid spaces', // trailing spaces
+        (v: string) => !(v.length < 5) || 'Court order number is invalid',
+        (v: string) => !(v.length > 20) || 'Court order number is invalid'
+      ]
+      this.$refs.courtNumRef.validate()
+    } else this.$refs.courtNumRef.resetValidation()
+  }
+
+  /** Emit court order number. */
+  @Watch('courtOrderNumber')
+  @Emit('emitCourtNumber')
+  private emitCourtNumber (): string { return this.courtOrderNumber }
```

Comment:
Oh that's clever (both a watcher and an emitter). 👍

Reply:
Seems to be a good way to keep the parent in sync with a local model. Useful for shared components, although i'm not sure what the other options are yet! Maybe there is a better way still??

Reply:
Not sure about other options, but for sure having a concise API between parent and component (ie, props + events) is good.